app1/admin.py

```diff
@@ -1,3 +1,19 @@
-from django.contrib import admin
+#from django.contrib import admin
 
 # Register your models here.
+from django.contrib import admin
+from .models import YourModel  # Import your model(s) here
```

Comment:
**Suggestion**: Use explicit imports instead of wildcard imports to improve code clarity and avoid potential namespace conflicts. <br> **Code Suggestion**: 
 ```
 - from .models import YourModel  # Import your model(s) here
 + from .models import YourModelName  # Replace 'YourModelName' with the actual model name
 ```